software-engineering-policies/SourceControl/SourceControlPolicy.md

```diff
@@ -9,34 +9,50 @@
 * Any source code found in any other source control method must first be migrated and proven to build via a CI pipeline.
 * All new source will host a primary branch called _trunk_ or _main_ over _master_.
 
+## Repository Setup
+
+Information about setting up your repository can be found [here](/software-engineering-policies/SourceControl/RepositorySetupPolicy.md)
+
 ## Access control
 
-Ensure that developers with BPSS clearance are only granted read access.  BPSS-cleared developers may temporarily be granted write access to a repository, provided suitable controls exist to prevent any change they make progressing to a live environment without review by a Security Cleared (SC) colleague.
+Access to a repo should be done through a team and not through an individual as to ensure that there is never a single point of failiure. Team access does not mean team ownership however.
+
+Ensure that developers with BPSS clearance are only granted read access. BPSS-cleared developers may temporarily be granted write access to a repository, provided suitable controls exist to prevent any change they make progressing to a live environment without review by a Security Cleared (SC) colleague.
```

Comment:
This is network specific, i.e. would not apply accross all network classifications.

Reply:
100%. I wrote this thinking more about github however - I need to reach out when I'm looking specifically at the AzDo stuff to figure out how you best do this so its documented correctly